tests/integration/test_array_split.py

```diff
@@ -39,11 +40,42 @@
 ]
 
 
+class TestArraySplitErrors:
+    """
+    this class is to test negative cases
+    """
+
+    def test_indices_negative(self):
+        # negative indices should be not accepted
+        ary = np.arange(10)
+        # In numpy, it raises ValueError,
+        # msg is "number sections must be larger than 0."
+        # while in cunumeric, it passed.
+        num.array_split(ary, -2)
```

Comment:
If this difference from numpy is unavoidable, then this test should maintain the cunumeric behavior explicitly (i.e. there should be an `assert` for the value returned)

Reply:
I updated its code to match numpy.